pkgs/top-level/all-packages.nix

```diff
@@ -20519,6 +20519,8 @@ with pkgs;
 
   protobuf = protobuf3_19;
 
+  protobuf3_21 = callPackage ../development/libraries/protobuf/3.21.nix { };
+  protobuf3_20 = callPackage ../development/libraries/protobuf/3.20.nix { };
```

Comment:
Do we actually need both or is 3.21 enough?

Reply:
I would prefer both, as you may need to introduce a specific version for some dev environments.